cli/kubectl_kadalu/storage_add.py

```diff
@@ -38,17 +38,40 @@ def storage_add_args(subparsers):
         default=[],
         action="append"
     )
+    parser_add_storage.add_argument(
+        "--external",
+        help="Storage from external gluster, Example: --external gluster-node:/gluster-volname",
+        default=[],
+        action="append"
```

Comment:
If multiple external nodes are not specified then `action="append"` can be removed and `default=""`. (Below it is checked that length == 1)

Reply:
Ok! makes sense.